access/client.go

```diff
@@ -62,7 +62,7 @@ type Client interface {
 	GetBlockByHeight(ctx context.Context, height uint64) (*flow.Block, error)
 
 	// GetCollection gets a collection by ID.
-	GetCollection(ctx context.Context, colID flow.Identifier) (*flow.Collection, error)
+	GetCollection(ctx context.Context, colID flow.Identifier) (*flow.LightCollection, error)
```

Comment:
I'm not sure if it is a good idea to change type's name as it might break code that depends on this. 
However, we want to do this to differentiate between light & full collection.

Reply:
reverted renaming